packages/nimble-components/src/rich-text-mention/base/models/mention-validator.ts

```diff
@@ -86,6 +86,7 @@ export class RichTextMentionValidator<
         );
     }
 
+    // TODO move this to child class like done for table
```

Comment:
What's the plan for addressing this TODO? An in-work part of this PR or a follow up?

Reply:
That was a todo I added. I had updated the pattern in the table to make the child classes do the supportedMappingElements check for some typing reasons. 

It's a small change if we can go ahead and do it to have alignment in the pattern. See here: https://github.com/ni/nimble/pull/1656/files#diff-3566b942a10cfb5f154b500ba99f0cd6362cd60a5247840f499a37a8d12f3220R47

Where:
- we stop passing `supportedMappingElements` to the base class
- the child class just has it's own implementation `validateMappingTypes`

Reply:
Moved `validateMappingTypes` to child class.